modules/spring-config/src/main/java/org/mule/runtime/config/internal/ModuleDelegatingEntityResolver.java

```diff
@@ -107,7 +108,12 @@ public InputSource resolveEntity(String publicId, String systemId) throws SAXExc
     }
     if (inputSource == null) {
       if (checkedEntities.get(systemId) != null) {
-        throw new MuleRuntimeException(createStaticMessage("Can't resolve %s %s", publicId == null ? "" : publicId, systemId));
+        String loadedExtensions = String.join(", ", extensions.stream().map(e -> e.getName()).collect(Collectors.toList()));
```

Comment:
this is not enough. Things like ApiKit or other plugins will not be registered here